content/router/lazy-load-feature-modules.md

```diff
@@ -41,7 +41,9 @@ This means that the `UsersModule` will be added to the main bundle. The main bun
 // app.routing.ts
 const routes: Routes = [
   ...
-  {path: 'users', loadChildren: '../users/usersModule#UserModule'}
+  {
+    path: 'users',
+    loadChildren: () => import('../users/usersModule').then(m => m.UsersModule)}
```

Comment:
Do you mind formatting this so that the curly brace at the end is aligned with its opening brace? Here's what I mean:
```
{
    path: 'users',
    loadChildren: () => import('../users/usersModule').then(m => m.UsersModule)
}
```

Reply:
change made